source/includes/fundamentals/code-snippets/Filters.java

```diff
@@ -123,18 +133,21 @@ private void bitsComparison() {
 
     private void geoWithinComparison() {
         // begin geoWithinComparison
+        // Creates a Polygon to filter the results
         Polygon square = new Polygon(Arrays.asList(new Position(0, 0), 
                                             new Position(4, 0), 
                                             new Position(4, 4), 
                                             new Position(0, 4),
                                             new Position(0, 0)));
    
+        // Matches documents with "coordinates" values that fall within the Polygon filter
```

Comment:
Suggestion:

I think the correct terminology would be "query filter". Another way to word this could be "the polygon passed as the filter parameter"

```suggestion
        // Prints documents that contain "coordinates" values that are all within the bounds of the polygon passed as the filter parameter
```